src/main/java/com/exasol/containers/status/ContainerStatusCache.java

```diff
@@ -0,0 +1,108 @@
+package com.exasol.containers.status;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Cache that remembers the container status across multiple test runs when the container is reused.
+ */
+public class ContainerStatusCache {
```

Comment:
How about writing the file inside of the container. Then the file would be deleted with the container

Reply:
Thought about this, but that has a couple of downsides:

* speed (uses `execInContainer` via HTTP)
* non-standard access (loss of convenience of file operations)
* testability
* temporal coupling (works only after the container is initialized)

All in all I found it better to live with the small temp files than with the hassle of the problems listed above.